### PR TITLE
Feature/Profile/cCLK32-1085-Update-Phone-Number

### PR DESCRIPTION
**Pull request by Minhdn1**
## Prerequisite
- [x] Self-review :monocle_face:
- [x] No Crashes ☠️
---

### Root cause
- // Explain what you have done

### Solution
- // Explain how you fix it

### Impact
- // List of affected components/modules

### Screenshots (Optional)
- // Capture affected screens

### CODE REVIEW

File: ClearKeep/Scenes/Profile/Views/CountryCode.swift [hunk 1]
```diff
@@ -23,27 +23,20 @@ struct CountryCode: View {
...
-
+	
 	// MARK: Body
 	var body: some View {
-			content
-		.background(background)
+		content
+			.background(background)
+			.hiddenNavigationBarStyle()
+//			.onAppear(perform: { self.data = self.datas.countryCodes })
```
Comment (vmo-klinh) [inline ClearKeep/Scenes/Profile/Views/CountryCode.swift]: remove unused code

File: Podfile.lock [hunk 2]
```diff
@@ -545,6 +554,6 @@ SPEC CHECKSUMS:
   SwiftSRP: 98943ac8497e2e7fb1ab1b63f68810aefc28010c
   YapDatabase: 350ab72b61f9dcc32504594da1a52a93f5ef1476
 
-PODFILE CHECKSUM: 0079586f5b3730bc3c147af29befdbdfd411e895
+PODFILE CHECKSUM: 909fafabcf7c96c27e245825c29e5e7828cea7f9
```
Comment (vmo-klinh) [inline Podfile.lock]: don't push your lock file

Comment (Minhdn1) [reply Podfile.lock]: fix complete

File: ClearKeep/Scenes/Profile/Views/CountryCode.swift (hunk 1 above)
Comment (Minhdn1) [reply ClearKeep/Scenes/Profile/Views/CountryCode.swift]: fix complete

**Review by vmo-klinh — APPROVED** [review]

**Pull request merged**